sdk/core/azure-core/setup.py

```diff
@@ -65,5 +65,6 @@
     ],
     extras_require={
         ":python_version<'3.0'": ['azure-nspkg'],
+        ":python_version>='3.5'": ['contextvars'],
```

Comment:
Just one thing - looking at the docs for the backport, it looks like it doesn't support asyncio yet. Which makes the use of it questionable.

Reply:
Oops. Then: https://github.com/Azure/azure-sdk-for-python/issues/6622